tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -58,10 +58,6 @@ test.describe( 'Set up accounts', () => {
 			)
 		).toBeVisible();
 
-		await expect(
-			page.getByRole( 'button', { name: 'Connect' } ).first()
-		).toBeEnabled();
-
 		const wpAccountCard = setUpAccountsPage.getWPAccountCard();
 		await expect( wpAccountCard ).toBeEnabled();
 		await expect( wpAccountCard ).toContainText( 'WordPress.com' );
```

Comment:
After removing the earlier assertion about the button being enabled, we should add it back here using the new `getWPAccountCard()` helper:

```suggestion
		await expect( wpAccountCard ).toContainText( 'WordPress.com' );
		await expect( wpAccountCard.getByRole( 'button' ) ).toBeEnabled();
```